lib/streamlit/report_thread.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         self.uploaded_file_mgr = uploaded_file_mgr
         # set_page_config is allowed at most once, as the very first st.command
         self._set_page_config_allowed = True
+        self.dg_stack = []
```

Comment:
nit: would copy over this comment so it's clear what a dg_stack is for:

"# Stack of DGs used for the `with` block. The current one is at the end."